triples-generator/src/main/scala/ch/datascience/triplesgenerator/eventprocessing/triplesgeneration/renkulog/Commands.scala

```diff
@@ -96,6 +103,11 @@ private object Commands {
         %%("git", "checkout", commitId.value)(repositoryDirectory)
       }.map(_ => ())
 
+    def checkoutCurrent(repositoryDirectory: Path): IO[Unit] =
+      IO {
+        %%("git", "checkout", ".")(repositoryDirectory)
+      }.map(_ => ())
```

Comment:
Could you maybe replace all `map(_ => ())` with `void`. I guess just in this file :)